tdrs-backend/tdpservice/settings/common.py

```diff
@@ -370,6 +385,35 @@ class Common(Configuration):
 
     ENABLE_DEVELOPER_GROUP = True
 
+    # XMS CONFIG
+    '''
+    "issuer":"https://int.xms.hhs.gov/",
+    "authorization_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/authorize",
+    "token_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/token",
+    "userinfo_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/userinfo",
+    "jwks_uri":"https://int.xms.hhs.gov/ABaaS/oidc/certificates"
+    '''
+    XMS_ACR_VALUES = os.getenv(
+        'ACR_VALUES',
+        'http://idmanagement.gov/ns/assurance/ial/1'
+    )
+    XMS_AUTHORIZATION_ENDPOINT = 'https://int.xms.hhs.gov/ABaaS/oidc/authorize'
+
+    XMS_CLIENT_ASSERTION_TYPE = os.getenv(
```

Comment:
we can remove this, xms doesn't require an assertion type. They handle this part of login.gov process for us